mlrun/utils/helpers.py

```diff
@@ -1266,6 +1266,36 @@ def is_legacy_artifact(artifact):
         return not hasattr(artifact, "metadata")
 
 
+def get_format_run(run: list, with_project=False):
```

Comment:
1. why is run a list? perhaps a dict?
2. Id change the signature as it doesnt "get" anything really but resolved/formatting the run

```suggestion
def format_run(run: list, with_project=False) -> dict:
```